contrib/check-config.sh

```diff
@@ -158,6 +166,16 @@ flags=(
 )
 check_flags "${flags[@]}"
 
+check_zfs() {
+	if [ -c /dev/zfs ]; then
+		wrap_good "  - /dev/zfs" 'present'
+	else
+		wrap_bad "  - /dev/zfs" 'missing'
+	fi
+	echo "  - $(check_command zfs)"
+	echo "  - $(check_command zpool)"
+}
```

Comment:
If we do something like the `check_device` I mentioned above, this will be short enough that it could go back down in the block below and we wouldn't need this inconsistent `check_zfs` function. :smile: